apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx

```diff
@@ -34,7 +34,6 @@ export function ValidatorFormDetail({ validatorAddress, unstake }: ValidatorForm
         delegatedStakeDataResult,
         systemDataResult,
         commission,
-        isCommitteeMember,
     } = useGetStakingValidatorDetails({
```

Comment:
we could remove the `isCommitteMember` flag from the `useGetStakingValidatorDetails` hook because looks to me like its not used anywhere. If it is used somewhere ,and I missed it, than we should update the   `useGetStakingValidatorDetails` hook to use the new hook  `useIsValidatorCommitteeMember` to calculate this flag.